pyproject.toml

```diff
@@ -39,6 +39,7 @@ anthropic = "^0.29.0"
 docstring-parser = "^0.15"
 pydantic = ">=1.9,<3"
 curl_cffi = "0.6.2"
+google-generativeai = "0.6.0"
```

Comment:
set google-generativeai as optional, you can refer to how litellm was set in toml file